gelf.go

```diff
@@ -140,9 +140,16 @@ func NewCore(options ...Option) (_ zapcore.Core, err error) {
 		return nil, err
 	}
 
+	var ws zapcore.WriteSyncer = w
+
+	if len(conf.writeSyncers) > 0 {
+		writers := append([]zapcore.WriteSyncer{w}, conf.writeSyncers...)
```

Comment:
Please change declaration to full (var) instead of shorten (:=)

Reply:
Which of these do you prefer?

A:

```go
var writers []zapcore.WriteSyncer
writers = append([]zapcore.WriteSyncer{w}, conf.writeSyncers...)
ws = zapcore.NewMultiWriteSyncer(writers...)
```

B:

```go
ws = zapcore.NewMultiWriteSyncer(
	append([]zapcore.WriteSyncer{w}, conf.writeSyncers...)...,
)
```

Reply:
I prefer something like this

```go
var writers = append([]zapcore.WriteSyncer{w}, conf.writeSyncers...)
ws = zapcore.NewMultiWriteSyncer(writers...)
```

Reply:
Oh right, apologies. I didn't even consider that as an option 🤦‍♂️